hal/src/main/native/athena/REVPH.cpp

```diff
@@ -266,6 +268,81 @@ HAL_Bool HAL_GetREVPHClosedLoopControl(HAL_REVPHHandle handle,
   return false;  // TODO
 }
 
+void HAL_SetREVPHCompressorConfig(HAL_REVPHHandle handle,
```

Comment:
We'd want these added to the REVPH header, as well as a properly named version of the PH_compressor_config_t struct. I'm willing to do all the high level stuff, so just the HAL header is necessary.